Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -199,7 +199,7 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_default</key>
 			<false/>
 			<key>pfm_description</key>
-			<string>Save the Personal Recovery Key (PRK) to disk for the MDM Escrow Service to collect.</string>
+			<string>Save the Personal Recovery Key (PRK) to disk for the MDM Escrow Service to collect</string>
```

Comment:
Can we revert these non-functional changes?  I see no reason to have personal preferences around periods and spaces to be mixed with the core of what this change is, which is adding additional preferences to this manifest.

Reply:
i didn't manually make those changes. that's just the plist provided by xcreds on their github

Reply:
@dev-yeet can you roll them back though?

Reply:
^ yeah we don't want functional changes interspersed with non-functional.  If there are discrepancies w/ punctuation and spacing, let's leave that out for a separate PR.

Reply:
sorry I missed this, does this work?

Reply:
This fix looks good, thanks.